README.md

```diff
@@ -1 +1,27 @@
 # bit-docs-prettify
+
+A bit-docs plugin that makes source-code snippets in HTML prettier.
+
+Powered by https://github.com/google/code-prettify.
+
+## Quick Start
+
+Add this package as a dependency to your project:
+
+    npm install bit-docs-prettify --save
+
+Your `bit-docs` enabled project should now have syntax highlighting of source-code snippets.
+
+### What's Included
```

Comment:
I would move this to CONTRIBUTING.md (understanding this isn’t needed for using it, right?) and rename it either [_Project Organization_](https://github.com/donejs/donejs/blob/major/docs/guides/contributing.md#project-organization), [_Code Organization_](https://documentjs.com/docs/DocumentJS.guides.contributing.html#section_CodeOrganization), or [_File organization & responsibilities_](http://canjs.com/doc/guides/contributing/code.html#Fileorganizationandresponsibilities).